litestar/utils/signature.py

```diff
@@ -131,6 +131,24 @@ def _is_any_optional(origin_: Any, args_: tuple[Any, ...]) -> bool:
     return hints
 
 
+def expand_type_var_in_type_hint(type_hint: dict[str, Any], namespace: dict[str, Any] | None) -> dict[str, Any]:
```

Comment:
I think this could be moved into `litestar.utils.typing.`

Reply:
I have moved the files.